.circleci/config.yml

```diff
@@ -85,6 +85,9 @@ commands:
             cat\<< EOF > ~/.cargo/config.toml
             [patch.crates-io]
             shuttle-service = { path = "$PWD/service" }
+            shuttle-aws-rds = { path = "$PWD/services/aws-rds" }
```

Comment:
Nit: we now have two top-level directories that are very closely named: `service` and `services`.

Perhaps we should rename `services` to `plugins` or `builders`?